### DISCUSSION TITLE
trigger method return value in v7

### DISCUSSION THREAD

**Discussion by markokrizan** [react-hook-form/react-hook-form#6475] [Q&A] [answered]
Hi,

The V6 to v7 migration guide mentions this change: 
```
- await trigger("test") // Returns true|false
+ await trigger("test") // Returns void
```
But it seems like the method still returns a `Promise<boolean>`. What are the plans for `trigger` method signature in the future - will the boolean return value going to be removed in future updates?

**Comment by bluebill1049** [comment, marked as answer]
They was rolled back because many users requested for it.

**Reply by groupsky** [reply]
is it possible to remove that from the migration guide?
